src/main/java/com/williamfiset/algorithms/search/QuickSelect.java

```diff
@@ -0,0 +1,39 @@
+package com.williamfiset.algorithms.search;
+
+public class QuickSelect {
+  public static int partition(int[] array, int start, int end) {
+    int pivot = start;
+    int temp;
+    while (start < end) {
+      while (start <= end && array[start] < array[pivot]) start++;
+
+      while (start <= end && array[end] > array[pivot]) end--;
+
+      if (start > end) break;
+      temp = array[start];
+      array[start] = array[end];
+      array[end] = temp;
+    }
+    temp = array[end];
+    array[end] = array[pivot];
+    array[pivot] = temp;
+    return end;
+  }
+
+  public static int findKthLargest(int[] array, int k) {
```

Comment:
shouldn't this be the K'th smallest element?

Reply:
Oh, yes. You're right. Changed that